spec/visual/templates/lexer.erb

```diff
@@ -1,21 +1,33 @@
 <style type="text/css">
+  main {
+    float: left;
+    padding: 15px;
+    width: calc(100% - 225px);
+  }
   .legend {
-    width: 200px;
-    margin-right: 30px;
+    padding: 15px;
+    padding-left: 0;
     float: right;
   }
 </style>
 
-<section class="legend">
-  <h2>Legend</h2>
-  <pre class="codehilite"><%
-    Rouge::Token.each_token do |token|
-      %><span class="<%= token.shortname %>"><%= token.qualname %></span>
-<%      end
-  %></pre>
-</section>
-<h2>Highlighted</h2>
-<%= @highlighted %>
+<header id="page-header">
+  <h1><%= @lexer.class.tag %> lexer visual test</h1>
+</header>
+<main>
+  <header class="section-header"><h2>Highlighted</h2></header>
+  <%= @highlighted %>
+
+  <header class="section-header"><h2>Raw</h2></header>
+  <pre><%= @raw %></pre>
+</main>
 
-<h2>Raw</h2>
-<pre><%= CGI.escape_html(@sample) %></pre>
+<aside class="legend">
+  <header class="section-header"><h2>Legend</h2></header>
+  <pre class="codehilite">
+<% Rouge::Token.each_token do |token|
+  %><span class="<%= token.shortname %>"><%= token.qualname %></span>
+<% end %>
+  </pre>
+</aside>
+<div class="clearfix"></div>
```

Comment:
Do we still need clearfix hacks? Genuine question.

Reply:
I'll check to confirm.. If required, I'll comment here. Else I'll push a commit.

Reply:
I confirm that the clearfix hack is needed since both the `<main>` and `<aside>` are floated blocks.

Reply:
Gotcha. Thanks for confirming.